client_test.go

```diff
@@ -523,6 +524,34 @@ func TestClientHeartbeat(t *testing.T) {
 
 		wg.Wait()
 	})
+
+	t.Run("timerReset panic", func(t *testing.T) {
```

Comment:
It looks like there are no assertions in this test. Is it checking that `client.resetActivityTimer()` doesn't panic?

Reply:
My bad, that was a leftover of my experiments 😅 I refactored the test to make a bit more sense.